Day04/Module11UnderstandingSequelize/Controllers/employeeController.js

```diff
@@ -0,0 +1,29 @@
+
+
+const getAllEmployees = async (req,res)=>{
```

Comment:
Unused method in controller, please avoid committing empty methods

Reply:
Updated